jicofo-common/src/main/java/org/jitsi/impl/protocol/xmpp/log/PacketDebugger.java

```diff
@@ -66,13 +67,17 @@ public PacketDebugger(XMPPConnection connection, @NonNull String id)
     @Override
     public void onIncomingStreamElement(TopLevelStreamElement streamElement)
     {
-        logger.debug(() -> "RCV PKT (" + id + "): " + streamElement.toXML());
+        logger.debug(
+                () -> "RCV PKT (" + id + "): " + XmlStringBuilderUtil.Companion.toStringOpt(streamElement.toXML())
+        );
     }
 
     @Override
     public void onOutgoingStreamElement(TopLevelStreamElement streamElement)
     {
-        logger.debug(() -> "SENT PKT (" + id + "): " + streamElement.toXML());
+        logger.debug(
+                () -> "SENT PKT (" + id + "): " + XmlStringBuilderUtil.Companion.toStringOpt(streamElement.toXML())
```

Comment:
If your method in jitsi-xmpp-extensions were on `Element` rather than `ExtensionElement` this could just use the non-companion flavor of `toStringOpt` -- is it worth it?  (I think it clearly would be if this were Kotlin, but it's less obvious for Java.)

Reply:
I noticed this was awkward, but I didn't think it's worth another jitsi-xmpp-extensions update